source/mainnet/net/index.rst

```diff
@@ -44,7 +44,8 @@
 
    guides/developer-page
    nodes/node-requirements
-   Smart contracts <https://developer.concordium.software/en/mainnet/smart-contracts/index.html>
+   Smart Contracts v1 - Testnet only <https://developer.concordium.software/en/mainnet/smart-contracts/index.html>
```

Comment:
I don't think this is how you want to link it, since when developing locally, it would navigate to the released version.
A downside to this is, that the whole smart contract table of content will appear in the sidebar
```suggestion
   Smart Contracts v1 - Testnet only <../smart-contracts/index>
```